kod/object/active/holder/room.kod

```diff
@@ -1906,6 +1906,12 @@ messages:
          propagate;
       }
 
+      % Break trance if the user is in one
+      if IsClass(what,&User) AND Send(what,@CheckPlayerFlag,#flag=PFLAG_TRANCE)
```

Comment:
You don't need to check the class.. the code just about this does it.

Reply:
I was hoping you'd comment on that. I did so because of the `plActive` loop that follows checking for class and thought whoever wrote that knew something I didn't.

Reply:
That's different---that's checking each *other* active object in the room to see if there are any other users still in the room.